### ISSUE TITLE
Tray icon not showing on pop os

### ISSUE THREAD

**Issue by freesig** [holochain/holoscape#43]
The tray icon is not displaying on popOS 19.10
## Steps to reproduce
1. Run holoscape with `npm i && npm start`.
2. Holoscape pop up asking which sim2h.
3. Holoscape run successfully
## Result
There's no tray icon so I can't actually use it.

I tried `sudo apt install libappindicator-dev` but that didn't work.

**Comment by freesig** [comment]
This fixes it. Maybe we should document it?
https://pop.system76.com/docs/status-icons/

**Comment by zippy** [comment]
Yes, that'sthe instructions I used to fix it too.  I think we should document but also probably need some other non system tray menu option too.
